docs/buttons/LightBgState.tsx

```diff
@@ -0,0 +1,240 @@
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  tableCellClasses,
+  TableContainer,
+  TableHead,
+  TableRow,
+  useTheme,
+} from '@mui/material';
+import { Button, Typography } from '@ui-kit-2022/components';
+import React from 'react';
+
+interface Props {
+  variantType?: 'PRIMARY' | 'SECONDARY' | 'TERTIARY';
+}
+
+const LightBgState: React.FC<Props> = ({ variantType }) => {
+  const theme = useTheme();
```

Comment:
If we use palette or other component in the theme, we can do `const {palette} = useTheme()` just save some writing time.